proto/kuksa/val/v1/val.proto

```diff
@@ -37,6 +37,8 @@ service VAL {
   // Set entries
   rpc Set(SetRequest) returns (SetResponse);
 
+  rpc StreamedUpdate(stream StreamedUpdateRequest) returns (stream StreamedUpdateResponse);
```

Comment:
From my memory it was decided to name it StreamedUpdate when we created the ticket (even though it was still unclear if now v1, v2 or whatever)

According to the ticket (https://etas-cloud.atlassian.net/browse/VAL-1320) and Databroker v2 API it will be named `provideStream` however I had the feeling that this does not fit very well for v1. In Databroker v2 API `set` is also named `provideOnce` therefore better fitting altogether. In the ticket it is also mentioning provideStream but I don't see this as a hard requirement.

However, I go with whatever you prefer.

Reply:
I think we have come to a consensus on what to call it for V2, but for v1 I really don't know what to call it.
For sdv is  StreamDatapoints, for v2 ProvideStream, for v1 I also think it was StreamedUpdate but not really sure.

Reply:
No hard opinion, just my 2ct; I don"t think this will need to be written very often, so a "long" (and honestly it is not that long) name should be no issue. At least this is saying what it does sort of. Better than "setS" :D